cmd/cluster/core/create.go

```diff
@@ -91,16 +91,19 @@ type CreateOptions struct {
 }
 
 type PowerVSPlatformOptions struct {
-	ResourceGroup   string
-	Region          string
-	Zone            string
-	CloudInstanceID string
-	CloudConnection string
-	VPCRegion       string
-	VPC             string
-	VPCSubnet       string
-	Debug           bool
-	RecreateSecrets bool
+	ResourceGroup          string
+	Region                 string
+	Zone                   string
+	CloudInstanceID        string
+	CloudConnection        string
+	VPCRegion              string
+	VPC                    string
+	VPCSubnet              string
+	Debug                  bool
+	RecreateSecrets        bool
+	UsePER                 bool
```

Comment:
These fields are increasing, lets start adding comments for these fields to have better understanding